dataactbroker/handlers/fileHandler.py

```diff
@@ -547,7 +547,12 @@ def addJobInfoForDFile(self, upload_file_name, timestamped_name, submission_id,
             exc = ResponseException(str(e),StatusCode.CLIENT_ERROR,ValueError)
             return False, JsonResponse.error(exc, exc.status, url = "", start = "", end = "",  file_type = file_type)
         # Create file D API URL with dates and callback URL
-        callback = "http://{}:{}/v1/complete_generation/{}/".format(CONFIG_SERVICES["broker_api_host"], CONFIG_SERVICES["broker_api_port"],task_key)
+        if CONFIG_SERVICES["broker_api_port"] == 443:
+            # Use https
+            prefix = "https"
```

Comment:
It's super nit-picky, but if we can just call it `protocol` instead of `prefix` it'd be very clear (even though people can see the value and determine that) but prefix is also used in other places, for example when doing stuff with S3.